qiita_pet/webserver.py

```diff
@@ -39,7 +39,8 @@
 STATIC_PATH = join(DIRNAME, "static")
 TEMPLATE_PATH = join(DIRNAME, "templates")  # base folder for webpages
 _, RES_PATH = get_mountpoint('job')[0]
-COOKIE_SECRET = b64encode(uuid4().bytes + uuid4().bytes)
+COOKIE_SECRET = 'TEST' if qiita_config.test_environment else \
```

Comment:
It makes it so if you're in a testing environment you don't have to constantly re-login if you change something. I can add a comment.